lib/ember-test-helpers/test-module.js

```diff
@@ -192,7 +192,7 @@ export default Klass.extend({
     this.cache = this.cache || {};
     this.cachedCalls = this.cachedCalls || {};
 
-    var keys = Ember.keys(callbacks);
+    var keys = Object.keys(callbacks);
```

Comment:
Can you make this fall back to Ember.keys if Object.keys is not present? If we don't do that, we break older browser compat for Ember 1.13...